usr/src/lib/libumem/common/envvar.c

```diff
@@ -226,6 +227,10 @@ static umem_env_item_t umem_debug_items[] = {
 		"Enables writing all logged messages to stderr",
 		&umem_output,	2
 	},
+	{ "checknull",		"Private",	ITEM_FLAG,
+		"Reports if allocating from the heap would return null",
```

Comment:
This sentence seems inaccurate in two respects. The first is that it doesn't really report, it aborts by default. Secondly, umem has multiple backends so it's not strictly speaking an allocation from the heap. If I set umem to use the mmap backend, this'll still fire.